templates/accounts/account/email.html

```diff
@@ -1,7 +1,6 @@
 {% extends "account/base.html" %}
 
 {% load i18n %}
-{% load url from future %}
```

Comment:
Here's the final list of differences (I ignored any difference which came down to 'non-erroring whitespace'):

In account:
	email.html
		- possible bugfix in the allauth version, probably no need to copy
	password_reset.html
		- '' instead of ""
	password_reset_from_key.html
		- missing .
		- '' instead of ""
	password_set.html
		- '' instead of ""
	In messages:
		logged_in.txt
			- empty on ours?

Everything else is identical